src/pps/pps.proto

```diff
@@ -443,6 +443,7 @@ message InspectJobSetRequest {
 
 message ListJobSetRequest {
   bool details = 1;
+  map<string, bool> projects = 2;
```

Comment:
Might be worth adding a comment here about the semantics, but they are pretty obvious.